src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotChangeEventSource.java

```diff
@@ -805,6 +830,28 @@ public void markSnapshotDoneOnServer(YBPartition partition, YugabyteDBOffsetCont
 
   }
 
+  /**
+   * Last snapshot record of the last snapshot batch will have a unique marker that will be used for indentification.
+   * Set all the fields of the record's Checkpoint to max values and set snapshot key to 'LAST_SNAPSHOT_RECORD'
+   * @param recordCheckpoint the record's checkpoint of the last snapshot record
+   */
+  private void setIdentificationMarkerForLastSnapshotRecord(OpId recordCheckpoint) {
+    recordCheckpoint.setTerm(Long.MAX_VALUE);
+    recordCheckpoint.setIndex(Long.MAX_VALUE);
+    recordCheckpoint.setKey(LAST_SNAPSHOT_RECORD_KEY);
+    recordCheckpoint.setWriteId(Integer.MAX_VALUE);
+    recordCheckpoint.setTime(Long.MAX_VALUE);
+  }
+
+  private Boolean IsLastSnapshotRecordOfLastBatch(OpId opid) {
```

Comment:
We follow camel casing in Java, so the first letter of the method name would be in lower case i.e. `isLastSnapshotRecordOfLastBatch`